### DISCUSSION TITLE
How to setup prompt format?

### DISCUSSION THREAD

**Discussion by MilesQLi** [axolotl-ai-cloud/axolotl#751] [Q&A]
From the .yaml configuration file, it doesn't ask the expected prompt format.

**Comment by winglian** [comment]
In the README, under datasets where you specify the data you wish to train, you can specify a prompt type for each dataset individually. This gives you flexibility in mixing and matching your datasets.

**Comment by profesanti** [comment]
How is the prompt format under inference mode? For example, if I do a finetuning on Mistral-7B-v0.1  using Alpaca format, how is the prompt format on inference?

**Reply by profesanti** [reply]
I think that this script has the answer: https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/src/axolotl/prompters.py The format change particularly to each style.

**Reply by NanoCode012** [reply]
Fyi, inference is quite bare. It would use the regular instruct/alpaca format or allow you to pass the full raw input.
